hbw/app/javascript/packs/hbw/components/claiming/pop_up.jsx

```diff
@@ -46,6 +46,7 @@ class HBWPopUp extends Component {
   updateSubscription = () => {
     const { tab } = this.state;
     const { claimingTasks } = this.props;
+    this.setState({ fetched: false });
```

Comment:
Do it when clear task list: 
https://github.com/latera/homs/blob/af387633dd47e540f521cc08bbfa3cebad85afb2/hbw/app/javascript/packs/hbw/components/claiming/pop_up.jsx#L40